server/services/admin.ts

```diff
@@ -210,18 +217,105 @@ export = ({ strapi }: StrapiContext): IServiceAdmin => ({
     };
   },
 
+  //Find all reports
+  async findReports(
+    this: IServiceAdmin,
+    {...query}: AdminFindAllProps,
+  ): Promise<ToBeFixed> {
+    const {
+      _q,
+      orderBy,
+      pageSize = 10,
+      page = 1,
+      filters,
+    }: AdminFindAllQueryParamsParsed = parseParams<AdminFindAllQueryParamsParsed>(
+      query,
+    );
+
+    const defaultWhere = {
+      $or: [{resolved: false}, {resolved: true}],
```

Comment:
```suggestion
      resolved: { $notNull: true },
```

See more: https://docs.strapi.io/developer-docs/latest/developer-resources/database-apis-reference/query-engine/filtering.html#notnull